packages/tools/devtools/devtools-view/src/components/TelemetryView.tsx

```diff
@@ -197,10 +255,12 @@ interface FilteredTelemetryViewProps {
 	 *
 	 */
 	telemetryEvents: ITimestampedTelemetryEvent[];
+	setIndex: React.Dispatch<React.SetStateAction<number | undefined>>;
```

Comment:
Nit: Could we add docs to these. We should also note what `undefined` means for both.